src/config.c

```diff
@@ -557,7 +551,7 @@ bool Config_Option_set(Config_Option_Field field, const char *val) {
 		return false;
 	}
 
-	if(config.cb) config.cb(field);
+	if(config.cb) return config.cb(field);
 
 	return true;
```

Comment:
```
if(config.cb) config.cb(field);
return true;
```
The new value is already set...
the only way i would return the value from the callback is if i'm supporting rollback of to the previous configuration